src/vs/workbench/contrib/preferences/browser/preferencesActions.ts

```diff
@@ -47,8 +47,9 @@ export class OpenSettings2Action extends Action {
 		super(id, label);
 	}
 
-	run(event?: any): Promise<any> {
-		return this.preferencesService.openSettings(false, undefined);
+	run(args: any): Promise<any> {
```

Comment:
The implementation refers to commit https://github.com/Microsoft/vscode/commit/e5c1c2258e421db353c2a2ee1a04da31837a7e77#diff-dfd9031c00a144c2364595f331243744

Reply:
I think we already support this for settings.

https://github.com/microsoft/vscode/blob/6eaf748366a48a5b6f9645cfa5e2257f51a027a6/src/vs/workbench/contrib/preferences/browser/preferences.contribution.ts#L230-L240

So not sure if you want to do this for all commands. @roblourens FYI